File: muchos-motors-front/src/app/shared/models/InventoryLog.ts
import { Customer } from './Customer';
import { InventoryCarPartLog } from './InventoryCarPartLog';
import { OrderItem } from './OrderItem';

export class InventoryLog {
  inventoryLogId: number | undefined;
  createdBy: string | undefined;
  createdDate: Date | undefined;
  isValid: boolean | undefined;
  inventoryCarPartLogs: InventoryCarPartLog[] | undefined;
}

File: muchos-motors/muchos-motors-api/Models/InventoryLog.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace muchos_motors_api.Models
{
    public class InventoryLog
    {
        [Key]
        public int InventoryLogId { get; set; }
        public string CreatedBy { get; set; }
        public DateTime CreatedDate { get; set; }
        public bool IsValid { get; set; }

    }
}
